recipes-multimedia/alsa/alsa-utils/hardkernel-odroidn2plus/90-alsa-restore.rules

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
Move this file directly to `recipes-multimedia/alsa/alsa-utils/90-alsa-restore.rules` if we will need some board specific files will add a directory for a board.

I'll check on multiple board to see if it works

Reply:
Ok, will do that tomorrow